examples/2d/2d_viewport_to_world.rs

```diff
@@ -77,6 +77,11 @@ fn controls(
     }
 
     if let Some(viewport) = camera.viewport.as_mut() {
+        // Reset viewport size on window resize
+        if viewport.physical_size.x > window_size.x || viewport.physical_size.y > window_size.y {
+            viewport.physical_size = (window_size.as_vec2() * 0.75).as_uvec2();
```

Comment:
Why 0.75? This needs a bit more explanation to explain why this is the right fix.